src/org/omegat/core/segmentation/datamodels/MappingRulesModel.java

```diff
@@ -89,7 +90,13 @@ public void setValueAt(Object aValue, int rowIndex, int columnIndex) {
         MapRule maprule = srx.getMappingRules().get(rowIndex);
         switch (columnIndex) {
         case 0:
-            maprule.setLanguage((String) aValue);
+            String target = (String) aValue;
+            String code = LanguageCodes.getLanguageCodeByName(target);
+            if (code != null) {
+                maprule.setLanguage(code);
+            } else {
+                maprule.setLanguage(target);
+            }
```

Comment:
A code here is for fixing the issue which  OmegaT produce a CONF/SRX file with localized language name.

Reply:
Works better but now if I close OmegaT and open it again, names are in english.
I will search whenever I can find a method to keep localization in the UI without having it in the file

Reply:
`getValueAt` is what you look for

Reply:
Yes, just pushed a correction to this branch 
Seems to work, now if I remove a rule set and create a new one still with name in french, setValueAt makes the correct correspondance between what I type and the code